test/dotnet-new.IntegrationTests/DotnetNewTestTemplatesTests.cs

```diff
@@ -0,0 +1,259 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Collections.Immutable;
+
+namespace Microsoft.DotNet.Cli.New.IntegrationTests
+{
+    public class DotnetNewTestTemplatesTests : BaseIntegrationTest
+    {
+        private readonly ITestOutputHelper _log;
+
+        private static readonly ImmutableArray<string> SupportedTargetFrameworks =
+        [
+            "net10.0",
+        ];
+
+        private static readonly (string ProjectTemplateName, string ItemTemplateName, string[] Languages, bool SupportsTestingPlatform)[] AvailableItemTemplates =
+        [
+            ("nunit", "nunit-test", Languages.All, false),
+            ("mstest", "mstest-class", Languages.All, false),
+        ];
+
+        private static readonly (string ProjectTemplateName, string[] Languages, bool RunDotnetTest, bool SupportsTestingPlatform)[] AvailableProjectTemplates =
+        [
+            ("nunit", Languages.All, true, false),
+            ("xunit", Languages.All, true, false),
+            ("nunit-playwright", new[] { Languages.CSharp }, false, false),
+        ];
+
+        public DotnetNewTestTemplatesTests(ITestOutputHelper log) : base(log)
+        {
+            _log = log;
+        }
+
+        public static class Languages
+        {
+            public const string CSharp = "c#";
+            public const string FSharp = "f#";
+            public const string VisualBasic = "vb";
+            public static readonly string[] All =
+                [CSharp, FSharp, VisualBasic];
+        }
+
+        private class NullTestOutputHelper : ITestOutputHelper
+        {
+            public void WriteLine(string message) { }
+
+            public void WriteLine(string format, params object[] args) { }
+        }
+
+        static DotnetNewTestTemplatesTests()
+        {
+            string templatePackagePath = Path.Combine(
+                RepoTemplatePackages,
```

Comment:
You've confirmed that this is the live location in the build, correct? I think it is but just double checking it's not the stage 0 (global.json) one as that version of templates is usually a month behind.